standalone-metastore/metastore-server/src/main/java/org/apache/hadoop/hive/metastore/leader/StaticLeaderElection.java

```diff
@@ -33,9 +33,9 @@
  * If the value is empty, means no election happening and all instances are leaders,
  * otherwise the instance matching the configured hostname will be selected as a leader.
  */
-public class HostLeaderElection implements LeaderElection<String> {
+public class StaticLeaderElection implements LeaderElection<String> {
```

Comment:
In HiveConf, we are referring to this as a host leader election approach but in the implementation, we are saying static leader election. why this discrepancy? Or Am I confused?

Reply:
The host leader election is a "static" way to elect a leader, means once if host is chosen, the leadership cannot switch to other instances even if the instance is down or the host is crashing, so I would rename it to the `StaticLeaderElection`, compared to the dynamic leader elelction.